tests/integration/mixer/policy/ratelimiting_test.go

```diff
@@ -47,14 +48,16 @@ var (
 )
 
 func TestRateLimiting_RedisQuotaFixedWindow(t *testing.T) {
-	testRedisQuota(t, fixedWindowConfig, "ratings")
+	testRedisQuota(t, bookinfo.RatingsRedisRateLimitFixed, "ratings")
 }
 
 func TestRateLimiting_RedisQuotaRollingWindow(t *testing.T) {
-	testRedisQuota(t, rollingWindowConfig, "ratings")
+	testRedisQuota(t, bookinfo.RatingsRedisRateLimitRolling, "ratings")
 }
 
+/*
 func TestRateLimiting_DefaultLessThanOverride(t *testing.T) {
+	t.Skip("testing")
```

Comment:
why skip testing this?

I think this is failing because in quotarule config destination service is productpage?

I agree with using samples config in above 2 test cases but we should keep rest of the functions so that other custom configs like this defaultlessthanoverride and external requests ratelimiting can be added?

Reply:
Sorry, didn't mean to. Still working on this PR.